sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1596,7 +1602,7 @@ def _post_port_sfp_info_and_dom_thr_to_db_once(self, port_mapping, xcvr_table_he
         transceiver_dict = {}
         retry_eeprom_set = set()
 
-        is_warm_start = is_warm_reboot_enabled()
+        is_warm_start = is_syncd_restore_complete()
```

Comment:
@ryanzhu706 can you provide more details about this syncd's restore_count and `WRM_RESTART_ENALE_TABLE` ?

Reply:
Syncd restore count / any restore count ---

RC (restore count) is incremented when service goes through warm-reboot.
So for eg., in the case of 3 warm-reboots, the count will be 3.

When the count gets reset:
1. When cold reboot happens
2. When service performs cold restart - config reload, load minigraph, service crash.


WARM_RESTART_ENALE_TABLE:

Gets enabled by warm-reboot shutdown:
https://github.com/sonic-net/sonic-utilities/blob/f45d89602dd40821057712f35856cf97d6a6ab40/scripts/fast-reboot#L710C1-L711C1

Gets reset by warm-reboot finalizer:
https://github.com/sonic-net/sonic-buildimage/blob/9673e12d43514bc98499d5de3d479e4562ca5c41/files/image_config/warmboot-finalizer/finalize-warmboot.sh#L128